src/gt4py/next/iterator/transforms/power_unrolling.py

```diff
@@ -64,8 +64,7 @@ def visit_FunCall(self, node: ir.FunCall) -> ir.Expr:
                 # Nest target expression to avoid multiple redundant evaluations
                 for i in range(pow_max, 0, -1):
                     ret = im.let(
-                        f"power_{2**i}",
-                        im.multiplies_(f"power_{2 ** (i - 1)}", f"power_{2 ** (i - 1)}"),
+                        f"power_{2 ** i}", im.multiplies_(f"power_{2**(i-1)}", f"power_{2**(i-1)}")
```

Comment:
Ah, now I understand the format change. The difference was not in the trailing comma, which seems to be already present in the previous version but in the space around the `-` in the exponent of the `power_` expression. Feel free to put back the trailing comma if you prefer.

Reply:
I left the original one...